english/chocolateOrSleep/chocolateOrSleep.md

```diff
@@ -4,3 +4,5 @@ Would you like some more [chocolate](../search_for_chocolate/search_for_chocolat
 would you like to [sing](../sing/sing.md)?
 
 would you like to [play](../play-music/playmusic.md)?
+
+Or maybe you'd like to [drive the cat to your sisters house](../dop-person/dog.md)?
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.